### DISCUSSION TITLE
[Projects Beta] Insights button as actual hyperlink

### DISCUSSION THREAD

**Discussion by marcreichel** [community/community#19962] [Projects and Issues] [answered]
Currently, it is not possible to open the project insights in a new tab because the “Insights” button is not an actual HTML hyperlink.

To be able to do so, I would love to see the button rendered as an HTML hyperlink instead.

**Comment by chiedo** [comment, marked as answer]
This should be resolved now @marcreichel! Sorry for the delay here.

**Reply by marcreichel** [reply]
Thank you very much! Works like a charm 👍🏼👍🏼
